### DISCUSSION TITLE
guide books

### DISCUSSION THREAD

**Discussion by aimanndaus** [sanidhyy/duolingo-clone#30] [Ideas]
**Is your feature request related to a problem? Please describe.**

can you add new features like duolingo have guide book for each unit ?
i hope fully you can add this features thansk sanidyy

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

![image](https://github.com/user-attachments/assets/ba758944-aff2-49a5-ac50-38050d4e5e89)
![image](https://github.com/user-attachments/assets/1e5a1ca0-5f84-43d1-b73b-36bbc82192d0)

**Comment by aimanndaus** [comment]
change the button to continue to guidebook

**Reply by sanidhyy** [reply]
Hey! I'll check out this feature when I have time. Thanks for the suggestion!

**Reply by aimanndaus** [reply]
thanks sanidhyy i hope you have a good days and peace in healthy. i waiting on your update
